build-tools/scripts/TestApks.targets

```diff
@@ -346,6 +346,8 @@
   <Target Name="CheckBootTimes"
       DependsOnTargets="AcquireAndroidTarget;ReleaseAndroidTarget">
     <RunAndroidEmulatorCheckBootTimes
+        AndroidSdkDirectory="$(AndroidSdkDirectory)"
+        AvdHome="$(AvdManagerHome)\.android\avd"
```

Comment:
If you compare to the `<CreateAndroidEmulator/>` task:

https://github.com/xamarin/xamarin-android/blob/02ab8f723b6b8e327a3d8a6fd99192598ca94668/build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/CreateAndroidEmulator.cs#L16-L20

https://github.com/xamarin/xamarin-android/blob/02ab8f723b6b8e327a3d8a6fd99192598ca94668/build-tools/scripts/TestApks.targets#L55

Is there something we could do to make both of these tasks more similar?

Reply:
I've updated all three tasks to now use `$(ANDROID_PREFS_ROOT)`.